sdk/identity/identity-cache-persistence/samples-dev/persistentCachingInteractiveBrowserCredential.ts

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
Add copyright statements here. I'm surprised our format tool didn't catch this.

```suggestion
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

/**
```